src/CameraControls.ts

```diff
@@ -2610,7 +2610,7 @@ export class CameraControls extends EventDispatcher {
 		const side = _v3B.crossVectors( cameraDirection, this._camera.up );
 		// Then find the vector orthogonal to both this "side" vector and the "view" vector.
 		// This vector will be the new "up" vector.
-		this._camera.up.crossVectors( side, cameraDirection );
+		this._camera.up = _v3C.crossVectors( side, cameraDirection ).normalize();
```

Comment:
Could this be:
```suggestion
		this._camera.up.crossVectors( side, cameraDirection ).normalize();
```
`crossVectors` returns `this`, which is the current vector3.

Reply:
Yup - everything seems to work with that. I'll update the PR.